src/plugins/discover/public/plugin.ts

```diff
@@ -1,31 +1,6 @@
 /*
+ * Copyright OpenSearch Contributors
```

Comment:
seems like keeping the old license makes sense here.

Reply:
I modified it since this file will change significantly by the end of the project.